Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
   "sdk/storage/azure_storage_blob",
   "sdk/template/azure_template_core",
   "sdk/template/azure_template",
+  "sdk/template/azure_template_macros",
```

Comment:
Please don't merge in the future. Do a rebase on `upstream/main` or it'll keep pulling in changes I've already viewed in other PRs.

Reply:
As discussed offline..
> This is a change I've added as part of my PR. There are no changes that are pulled in from other PRs. 

That said, removed `azure_template_macros` package as discussed offline.